tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -1946,13 +1948,8 @@ class RemapperFuseMatMulWithBiasAndActivationTest : public RemapperTest {
   template <DataType DTYPE>
   void RunTest() {
     using ::tensorflow::ops::Placeholder;
-
-#if defined(INTEL_MKL) && defined(ENABLE_MKL)
-    std::vector<string> activations = {"Relu", "Relu6", "Elu", "Tanh",
-                                       "LeakyRelu"};
-#else
     std::vector<string> activations = {"Relu", "Relu6", "Elu", "LeakyRelu"};
-#endif
+    if (IsMKLEnabled()) activations.push_back("Tanh");
```

Comment:
Are you sure it's only enabled with MKL?  I believe I added this for CPU in commit 8cb3e0a4901bb7d5f62acc033220408ade3dc4e8.

Reply:
Good catch! Fixed it.

Reply:
Looks like removing the MKL check is causing the test to fail on GPU. I've fixed it by disabling the test on GPU.